### PR TITLE
Moving to the new colors

### PR DESCRIPTION
**Pull request by Duncid**
## Description
Cleaning colors for consistency and moving to new colors.

## Risk
UI risks mostly

### CODE REVIEW

File: front/tailwind.config.js [hunk 1]
```diff
@@ -150,91 +150,109 @@ module.exports = {
...
-          800: { DEFAULT: colors.blue[800], dark: colors.blue[200] },
-          700: { DEFAULT: colors.blue[700], dark: colors.blue[300] },
-          600: { DEFAULT: colors.blue[600], dark: colors.blue[400] },
-          500: { DEFAULT: colors.blue[500], dark: colors.blue[500] },
-          400: { DEFAULT: colors.blue[400], dark: colors.blue[600] },
-          200: { DEFAULT: colors.blue[200], dark: colors.blue[800] },
-          300: { DEFAULT: colors.blue[300], dark: colors.blue[700] },
-          100: { DEFAULT: colors.blue[100], dark: colors.blue[900] },
-          50: { DEFAULT: colors.blue[50], dark: colors.blue[950] },
+          DEFAULT: colors.blue[500],
```
Comment (JulesBelveze) [inline front/tailwind.config.js:217]: I guess we want it slate here?

**Review by JulesBelveze — COMMENTED** [review]
Let's keep the "white" variant for buttons mainly for the "Stop Generate" button. It otherwise is invisible when on top of markdown (as its background is transparent)

**Comment by Duncid** [comment]
@JulesBelveze let's just add a white background to "outline"!

**Review by JulesBelveze — APPROVED** [review]
LGTM 🔥

**Pull request merged**